kuma/static/styles/components/structure/login.scss

```diff
@@ -16,7 +16,7 @@ Header login link
         vertical-align: top; /* needed on mobile */
     }
 
-    > a:nth-child(n+2) {
+    > a, button:nth-child(n+2) {
```

Comment:
I would suggest moving the new `.logout` rule from `forms.scss` to this file, adding the `bidi-style` rules below to it, then removing this selector entirely.